src/Misc/PerformanceViewer/growableFloat32Array.ts

```diff
@@ -0,0 +1,87 @@
+const growthFactor = 1.5;
+const bytesPerItem = 32 / 8;
+
+/**
+ * A class acting as a growable float32array used in the performance viewer
+ */
+export class GrowableFloat32Array {
+    private _view: Float32Array;
+    private _length: number;
+
+    /**
+     * Creates a new GrowableFloat32Array with the desired capacity.
+     * @param _capacity The initial capacity you would like to set for the array.
+     */
+    constructor(private _capacity: number) {
+        const buffer = new ArrayBuffer(_capacity * bytesPerItem);
+        this._view = new Float32Array(buffer);
+        this._length = 0;
+    }
+
+    /**
+     * The number of items currently in the array.

```

Comment:
Byte or Item would be great to be in the prop name :-)

Reply:
changed to itemLength!